tests/e2e/pre_provisioning.go

```diff
@@ -0,0 +1,130 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"fmt"
+	"math/rand"
+	"os"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	powervscloud "github.com/ppc64le-cloud/powervs-csi-driver/pkg/cloud"
+	"github.com/ppc64le-cloud/powervs-csi-driver/tests/e2e/driver"
+	"github.com/ppc64le-cloud/powervs-csi-driver/tests/e2e/testsuites"
+	v1 "k8s.io/api/core/v1"
+	clientset "k8s.io/client-go/kubernetes"
+	"k8s.io/kubernetes/test/e2e/framework"
+)
+
+const (
+	defaultDiskSize   = 4
+	defaultVolumeType = powervscloud.VolumeTypeTier3
+	debug             = false
+	apiKeyEnv         = "IBMCLOUD_API_KEY"
+)
+
+var (
+	defaultDiskSizeBytes int64 = defaultDiskSize * 1024 * 1024 * 1024
```

Comment:
take unsigned int here 'uint64'  int64. some time goes to negative range

Reply:
Since this is a controller environment, we already know that the value for defaultDiskSize is 4. Do you still think that we need to change it to `unint64`?

Reply:
Agree.